src/views/Workspaces/Workspaces.js

```diff
@@ -3,13 +3,87 @@
 
 import React from 'react';
 import { AppBar } from '../../components/AppBar';
-// import { useNavigate } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom';
+import { Grid, Accordion, AccordionSummary, AccordionDetails, Typography } from '@material-ui/core';
+import { useTranslation } from 'react-i18next';
+import { ResourceCard } from '@cosmotech/ui';
+import { useWorkspaces } from './WorkspacesHook';
 
 const Workspaces = () => {
-  // const navigate = useNavigate();
+  const navigate = useNavigate();
+  const { t } = useTranslation();
+  const { workspacesList, organizationName, selectWorkspace } = useWorkspaces();
+  sessionStorage.removeItem('providedUrlBeforeSignIn');
+
+  const labels = {
+    openButtonWorkspaceCard: t('genericcomponent.workspaceselector.card.button.open', 'Open'),
+    titleNoWorkspacePacePlaceHolder: t(
```

Comment:
Typo: `WorkspacePace` , it appears several times in this file

Reply:
And placeHolder one more time (same on lines 24 and 28) :slightly_smiling_face: